src/core/meshcop/tlvs.hpp

```diff
@@ -51,6 +52,21 @@ using Thread::Encoding::BigEndian::HostSwap32;
 namespace Thread {
 namespace MeshCoP {
 
+enum
+{
+    kMeshCoPMessagePriority = Message::kPriorityHigh, // The priority for MeshCoP message
+};
+
+/**
+ * This function create Message for MeshCoP
+ *
+ */
+inline Message* NewMeshCoPMessage(Coap::CoapBase &aCoapBase, const Coap::Header &aHeader)
```

Comment:
I don't think this really fits in `tlvs.hpp`.  How about creating a new file called `meshcop.hpp`?